src/components/FlexibleForm/Control.js

```diff
@@ -29,9 +29,24 @@ const controls = {
 
 const getControl = controlType => components[controlType] || controls[controlType];
 const getValidation = validation => validation.map(val => validators[val]);
+const getActualName = (name, sectionNamespace, repeatableIndex) => {
+  const nsName = name && sectionNamespace ? name.split('**ns**').join(sectionNamespace) : name;
+
+  return nsName && repeatableIndex >= 0 ? nsName.split('##ri##').join(repeatableIndex) : nsName;
+};
+const getOptions = (options, sectionNamespace, intl) => {
+  return options.map(option => {
+    const lbl = option.label;
+
+    return {
+      value: option.value,
+      label: intl.formatMessage({ id: lbl && sectionNamespace ? lbl.split('**ns**').join(sectionNamespace) : lbl }),
```

Comment:
I see that 
```lbl && sectionNamespace ? lbl.split('**ns**').join(sectionNamespace) : lbl``` check pretty much the same as
```name && sectionNamespace ? name.split('**ns**').join(sectionNamespace) : name``` [here](https://github.com/folio-org/ui-data-import/pull/444/files#diff-6709362c6d184124da81079fe4cef458R33)

I think some util function would be helpful here.